requirements.txt

```diff
@@ -14,5 +14,9 @@ norbert
 tqdm
 termtables
 gpytorch
+ffmpeg-python
 importlib_resources
-pyloudnorm
\ No newline at end of file
+pyloudnorm
+sox==1.4.0b0
+numba==0.48
+soxbindings
```

Comment:
The `soxbindings` should move to a special way to install `nussl`: `pip install nussl[soxbindings]`, or `pip install -e ".[soxbindings]"` if installing from source. Do this in `setup.py`.